src/Server/sql/index.sql

```diff
@@ -882,6 +882,6 @@ insert into privileges (feature, name, description, quantity) values
     (100, 'Start chats', 'View new chat suggetions, and start new chats', 25),
     (200, 'Participate in chat experiments', 'Take part in chat experiments. Some experiments may require more karma', 50),
     (201, 'Impersonation', 'Start Impersonation chat experiment', 1000),
-    (300, 'More tags', 'Increased number of max profile tags', 500),
-    (400, 'Send links', 'Send and automatically highlight links in chats', 3000),
-    (500, 'Send images', 'Send pictures in chats', 10000);
\ No newline at end of file
+    (300, 'More tags', 'Increased number of max profile tags', 250),
+    (400, 'Send links', 'Send and automatically highlight links in chats', 444),
+    (500, 'Send images', 'Send pictures in chats', 1500);
```

Comment:
I think these new values are too low. What made you pick them? Also we can change the text of send links to something like 'send markdown links' to reflect what it actualy is

Reply:
>I think these new values are too low

What about 3000 for images? 5000 would only allow the top 8 users (not including whoami) to use it. Even from those 8, 2 or 3 are currently inactive. 

>What made you pick them?

1500: 3000 was the previous value for links, so halved that. 

444: ratio 

> send markdown links

Sounds good. I'll change it.

Reply:
3000 sounds nice. we dont want too many people sending pictures cos of server space too :skull:

Reply:
My calculation for 444 was incorrect should have been 579. 

Anyway, with 3k in mind, here are the new values:

250
974
3000

The formula being:
`(x-250) / (3000 - x) = 2500 / 7000`